test/e2e/framework.go

```diff
@@ -74,10 +78,16 @@ type TestDataSummary interface {
 
 // NewFramework makes a new framework and sets up a BeforeEach/AfterEach for
 // you (you can write additional before/after each functions).
-func NewFramework(baseName string) *Framework {
+func NewDefaultFramework(baseName string) *Framework {
+	return NewFramework(baseName, 5, 10)
+}
+
+func NewFramework(baseName string, qps float32, burst int) *Framework {
```

Comment:
Could we create a params struct and pass that in with defaults?   I'm just thinking that would be cleaner as the configurable params grow.